.golangci.yml

```diff
@@ -10,7 +10,7 @@ linters:
     - govet
     - ineffassign
     - misspell
-    # - nakedret
+    - nakedret
```

Comment:
What is the benefit?

Reply:
It is a matter of choice, and deciding if we want to allow naked returns or not.

I (clearly) think we should not.

Naked returns IMHO are a very contentious language feature (https://github.com/golang/go/issues/21291 ).

They make code harder to read and understand, more confusing to newcomers (have to look back to the function signature to figure out what is being returned), and they are likely error prone, for example when err gets shadowed (named returns are already problematic in that regard when used with defer).

We only had two in the codebase.
Adding the linter will ensure we won't get any in the future.

WDYT?